src/freertos_drivers/common/Devtab.hxx

```diff
@@ -46,6 +46,9 @@ class FileSystem;
 class Notifiable;
 class DeviceBufferBase;
 
+#ifdef ESP32
```

Comment:
This seems like a "make it build" hack.  If we don't use POSIX, we should find out why it is getting pulled into the build.

Reply:
yes, it was in a lot of ways. I'll be dropping this once I solve the includes.